Edited qshutdown/README because of the removal of the DBus method "RequestShutdown" on (most) systems using Gnome Shell
Index: shutdown-qapps-1.6.5/qshutdown/README
===================================================================
--- shutdown-qapps-1.6.5.orig/qshutdown/README	2013-04-20 13:48:44.594032540 +0200
+++ shutdown-qapps-1.6.5/qshutdown/README	2013-04-20 13:48:44.666032536 +0200
@@ -56,6 +56,11 @@
 the rights in the sudoers, but it still doesn't work, please specify
 your own command in the preferences and try if "sudo shutdown -hP" now works.
 
+For most systems using the Gnome Shell a certain DBus method has been removed,
+thus a shutdown dialog will be likely to appear if qshutdown's methods have
+been set to automatic or to Gnome session. If you do not want this
+shutdown dialog to appear, please use ConsoleKit or something else.
+
 
 
 HINTS ON USAGE:
